docs/DeveloperGuide.md

```diff
@@ -326,10 +332,23 @@ The `RemoveCommand` class is responsible for removing the values within fields o
 The following sequence diagram shows how the `RemoveCommand` is executed:
 ![](UML/RemoveCommand_Sequence_Diagram.png)
 
-### Product scope
+## Product scope
 ### Target user profile
```

Comment:
Do we have an index page for sections from this section onwards?

Reply:
Hi Ting Hui, what do you mean by index page? If you are referring to the section header above, I have added the hyperlink references above.